### PR TITLE
Add InputError component

### LINKED ISSUE #130: Component - Input Error
# Description

Create the [Input Error component](https://www.figma.com/file/tXs6IdvpuX4jCLBWbJx0C6/Components-and-Patterns?node-id=34%3A82). The component should follow the format and style guide shown in the wireframe.

- Create a InputError component.
- The component should have two props: a boolean prop that when triggered shows the error message, and a string for the error message.
- The component should be just a paragraph tag styled like shown in the design.
- There should be a story created for it as well.

[Input Documentation](https://www.figma.com/file/5w15OeN9YiOR4UX0XKW7so/Documentation?node-id=105%3A268)

# Assessment Criteria
  - [ ] Component has all states and follows the format and style guide.
  - [ ] Meets accessibility standards
  - [ ] Responsive (mobile, tablet, laptop)

### PR DESCRIPTION
**Pull request by tristan-orourke**
Resolves #130.

Also added red color with opacities to h2 config.

### CODE REVIEW

File: admin/resources/js/components/form/InputError.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+}) => {
+  return isVisible ? (
+    <p
+      data-h2-display="b(inline-block)"
+      data-h2-radius="b(s)"
+      data-h2-bg-color="b(red[.1])"
+      data-h2-padding="b(all, xs)"
+      data-h2-color="b(red)"
+      role="alert"
+    >
```
Comment (mnigh) [inline admin/resources/js/components/form/InputError.tsx:15]: might be a good idea to add `data-h2-font-*` properties such as `data-h2-font-family="b(sans)"` to match the design, unless styles will be inherited from a planned higher component?

**Review by mnigh — COMMENTED** [review]
nice work: just one question about font styles for the component.

File: admin/resources/js/stories/InputError.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { Story, Meta } from "@storybook/react";
+import InputError from "../components/form/InputError";
+
+const meta: Meta = {
+  title: "Input Error",
```
Comment (mnigh) [inline admin/resources/js/stories/InputError.stories.tsx:6]: might this be better named `Components/Input Error` for maintaining a [hierarchy](https://storybook.js.org/docs/riot/writing-stories/naming-components-and-hierarchy)? Or `Hydrogen/Input Error` and then changing Input Label to match?

Comment (tristan-orourke) [reply admin/resources/js/stories/InputError.stories.tsx:6]: I'll change this to match InputLabel

File: admin/resources/js/components/form/InputError.tsx (hunk 1 above)
Comment (tristan-orourke) [reply admin/resources/js/components/form/InputError.tsx]: I assume the inputs should use the same font as the rest of the app?

However, I did just notice that the designs seem to use a different font size for the label and error.

Comment (tristan-orourke) [reply admin/resources/js/components/form/InputError.tsx]: I added default font family in #220

**Review by mnigh — APPROVED** [review]
👍

**Pull request merged**